lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -264,21 +266,25 @@ defmodule ExUnit.Runner do
             end
           end)
 
-        send(parent, {self(), :test_finished, %{test | time: us}})
+        send(parent_pid, {self(), :test_finished, %{test | time: us}})
         exit(:shutdown)
       end)
 
-    timeout = get_timeout(test.tags, config)
-    test = receive_test_reply(test, test_pid, test_ref, timeout)
+    reply_opts = %{
+      pid: test_pid,
+      ref: test_ref,
+      parent_pid: parent_pid,
+      timeout: timeout
+    }
+
+    test = receive_test_reply(test, reply_opts)
 
     exec_on_exit(test, test_pid, timeout)
   end
 
-  defp generate_test_seed(%{seed: seed}, %ExUnit.Test{module: module, name: name}) do
-    :rand.seed(@rand_algorithm, {:erlang.phash2(module), :erlang.phash2(name), seed})
-  end
+  defp receive_test_reply(test, opts) do
```

Comment:
It should not be called `opts`, it should be called `config`, as it is a structured map.

Reply:
good to know, I didn't know that was the difference